### ISSUE TITLE
Sticker search displays incorrect result, but places correct symbol.

### ISSUE THREAD

**Issue by txcrew** [romannurik/Sketch-Stickers#29]
Found a potential bug that reveals itself when searching for a specific sticker, but displays the first sticker in a set of results.

In the below example, the user is searching `disabled` which should surface the result, `item disabled`. Instead, it shows the first result in the row of stickers (in the case of the below example it's the sticker labeled `logo`). However, when the user drags the sticker into the document, it's the correct symbol.

![screen](https://user-images.githubusercontent.com/42686094/82491864-e1b3db80-9aaa-11ea-807b-e09370016c97.gif)

These are the tags in the file

```
!StickerLibrary
title: My library

!StickerSection @Section1
title: First section
```

**Comment by romannurik** [comment]
Whoa weird. Does it persist if you clear your library cache? If it does, I'd love if you could privately share the file with me so I could test.

**Comment by txcrew** [comment]
Yeah, unfortunately, it still does it even after I disabled all other plugins, cleared the library cache, and relaunched Sketch. Here's a library it happens on (a little different than the example), just search red, green, or blue in the sticker sheet (and also try clearing the text string and typing a new one): https://send.firefox.com/download/e6da5701f12d9a0b/#qo2QIFsstZ3bVQZ-nzBsbg

**Comment by txcrew** [comment]
After further tests, the weird overlap only seems to happen in a `stickerSection` without a `sub-section`. Not sure if that will help pinpoint the issue, but I figured I'd share the finding. Here's hoping! :)

**Comment by romannurik** [comment]
Thank you so much for digging deeper! I'll take a look at a fix when I can.
